package.json

```diff
@@ -92,11 +94,13 @@
     "@babel/preset-env": "7.16.11",
     "@babel/preset-react": "7.16.7",
     "@babel/preset-typescript": "7.16.7",
+    "@senecacdot/eslint-config-telescope": "latest",
```

Comment:
Why do we need the `latest` version in here? Can't we specify the version of `@senecacdot/eslint-config-telescope`?

Reply:
I copied and pasted this line from somewhere in our code. But you're right, it should be 1.0.0. I will fix this by tomorrow morning. Thanks for catching this!